test/parallel/test-buffer-tojson.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const Buffer = require('buffer').Buffer;
```

Comment:
I generally prefer explicit code over implicit, at least in tests... mostly a personal preference than anything else.